packages/blade/src/components/Tabs/TabItem.native.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/no-unnecessary-type-assertion */
+import styled from 'styled-components';
+import get from 'lodash/get';
+import React from 'react';
+import type { TabItemProps, TabsProps } from './types';
+import { paddings } from './tabTokens';
+import { makeSpace } from '~utils';
+import BaseBox from '~components/Box/BaseBox';
+
+const StyledTabButton = styled(BaseBox)<{
+  size: TabsProps['size'];
+  autoWidth?: TabsProps['autoWidth'];
+  variant: NonNullable<TabsProps['variant']>;
+}>(({ theme, size, variant, autoWidth }) => {
+  return {
+    display: 'flex',
+    alignItems: 'center',
+    flexDirection: 'row',
+    width: autoWidth ? '100%' : undefined,
+    paddingTop: makeSpace(get(theme, paddings[variant].horizontal.mobile.top[size!])),
+    paddingBottom: makeSpace(get(theme, paddings[variant].horizontal.mobile.bottom[size!])),
+    paddingLeft: makeSpace(get(theme, paddings[variant].horizontal.mobile.left[size!])),
+    paddingRight: makeSpace(get(theme, paddings[variant].horizontal.mobile.right[size!])),
+  };
+});
+
+const TabItem = (_props: TabItemProps): React.ReactElement => {
```

Comment:
So internally we just loop over the TabItems and [extract](https://github.com/razorpay/blade/pull/1707/files#diff-867b33d70dec2cbec73a53d6f39a4cdfd13fdd6a35a9dfd194b07e967db4bdc2R46-R53) it's props. This component itself doesn't need to render anything. 

Since rendering is handled by the react-native-tab-view library's [renderTabBar](https://reactnavigation.org/docs/tab-view/#rendertabbar) etc APIs

Reply:
Write a comment in these files to avoid confusion?

Reply:
Done.